custom-tests.yaml

```diff
@@ -1500,7 +1500,11 @@ api:
       if (!constructor) {
         return false;
       }
-      var instance = new constructor();
+      var cnstResult = bcd.testConstructor(constructor);
```

Comment:
@vinyldarkscratch is this bit really needed, would just adding the `{iceServers: []}` argument not have sufficed? `RTCPeerConnection` has always been a thing you construct, it will never have thrown an exception with the kind of message that `bcd.testConstructor` looks for.

Reply:
I've sent https://github.com/foolip/mdn-bcd-collector/pull/1424.

Reply:
It does, actually -- in early versions of Firefox when Web RTC was enabled by a flag, the `RTCPeerConnection` constructor _was_ defined, but attempting to construct something with it returns an error saying it "is not a constructor", thus resulting in null values.

Reply:
I see, can you add a comment to the test?